### ISSUE TITLE
Beamer does not respect ordered list style

### ISSUE THREAD

**Issue by klapheke** [jgm/pandoc#3021]
If you take a simple markdown file with ordered lists:

```
a. this
b. that

* * * * *

i.  here
ii.  there
```

and convert to beamer (`pandoc -t beamer -o beamer.pdf beamer.md`), the ordered lists are all rendered as Arabic numerals. It looks like beamer doesn't use `labelenumi` to style lists.

I worked around this by adding the following in my preamble, but maybe there's a better way?

``` latex
\setbeamertemplate{enumerate item}{\expandafter\labelenumi}
\setbeamertemplate{enumerate subitem}{\expandafter\labelenumii}
\setbeamertemplate{enumerate subsubitem}{\expandafter\labelenumiii}
```

**Comment by jgm** [comment]
+++ Alex Klapheke [Jul 15 16 11:09 ]:
>   I worked around this by adding the following in my preamble, but maybe
>   there's a better way?
>
>\setbeamertemplate{enumerate item}{\expandafter\labelenumi}
>\setbeamertemplate{enumerate subitem}{\expandafter\labelenumii}
>\setbeamertemplate{enumerate subsubitem}{\expandafter\labelenumiii}

I'd consider adding this to the default template, but I'd
like to know first why beamer doesn't already do this (that
is: what could break?)

**Comment by klapheke** [comment]
Thanks for the quick response! I don't know enough about beamer to say, though `\setbeamertemplate` is a high-level command [documented in the manual](http://tug.ctan.org/macros/latex2e/contrib/beamer/doc/beameruserguide.pdf#page=169). The other thing that seems to work is to change `\def\labelenumi{...}` to `\setbeamertemplate{enumerate item}{...}`.

The way [prescribed by the manual](http://tug.ctan.org/macros/latex2e/contrib/beamer/doc/beameruserguide.pdf#page=112) is to use the syntax of the `enumerate` package:
```latex
\begin{enumerate}[a.]
    \item ...
\end{enumerate}
```

**Comment by mb21** [comment]
This seems to be fixed in pandoc 2.0.5
